2200056/문제1 5431민석이과제.py

```diff
@@ -0,0 +1,9 @@
+T = int(input())
+for t in range(1, T+1):
+    students = set([]) # 차집합 이용하기 위해서 set사용
+    N, K = map(int, input().split())
+    for i in range(1, N+1):
+        students.add(i) # 수강생 번호 담기
```

Comment:
```suggestion
    students = set(range(1, N + 1)) # 수강생 번호 담기
```
위와 같은 방법으로 `students`를 만들 수도 있습니다 😄 

추가로, **set은 자동 정렬 해주지 않아요!** 정렬된 것처럼 보일 뿐, 숫자가 커지면 순서가 섞이게 됩니다!